lib/resources/order-recipients.js

```diff
@@ -125,7 +139,7 @@ module.exports = orderRecipientFactory;
  * @typedef module:resources/orders-recipients~PiiRequestBody
  *
  * @property {string} id Member ID
- * @property {string} [accept_language] The member's preferred locale, in the format of Accept-Language
+ * @property {string} accept_language The member's preferred locale, in the format of Accept-Language
```

Comment:
So we'll need to outline in the CHANGELOG that this is another new required prop… I think this is actually a BREAKING CHANGE, unless the user is _not_ using PII. 

I've got to think this through a bit. Let's jump on a call.